### DISCUSSION TITLE
Google Maps looks for icons locally when Tailwind is installed and can't find them

### DISCUSSION THREAD

**Discussion by catherineluse** [tailwindlabs/tailwindcss#5785] [Help] [answered]
I'm trying to make a cluster of map markers as shown in this example https://www.npmjs.com/package/@googlemaps/markerclusterer

I used the `@fawmi/vue-google-maps` library to render the map in Vue. In normal Vue apps it works, but in my app with Tailwind CSS installed, I get a lot of console errors about how the icons cannot be found: `GEThttp://localhost:3000/images/m1.png`

![Screen Shot 2021-10-14 at 3 11 07 PM](https://user-images.githubusercontent.com/20599230/137402784-6fcc98d4-b910-43a9-9ba8-1e9a9a9de52d.png)

I also had to manually add CSS to make it so that the map would be large enough to read:

```
.vue-map {
    height: 500px;
}
```

It seems Tailwind overrode some default styles that affected Google Maps. What is the best practice to avoid that conflict? I'd like Google Maps to use its default icons and styles.

The source code is here https://github.com/catherineluse/vue-playground

**Comment by simonswiss** [comment, marked as answer]
Hey!

Very unlikely that Tailwind is the reason this happens.

I am not familiar with the `vue-google-maps`, but I noticed in your codebase (which I cloned) that the `style` attribute you pass to the `GMapMap` component is not being applied to the element with the `vue-map-container` class:

![image](https://user-images.githubusercontent.com/485747/137415966-48a60752-bdff-4d25-8b55-631d613523d0.png)

This is why you need to manually add a height to the map.

I can see in the CodeSandbox of `vue-google-map` demo that the style attribute is being applied there, with code that looks similar:

https://codesandbox.io/s/vue-3-google-maps-ygpr4?file=/src/components/Map.vue

![image](https://user-images.githubusercontent.com/485747/137416205-c953cdfc-3fb8-4832-a8e1-95ce57f976c3.png)

![image](https://user-images.githubusercontent.com/485747/137417099-fb9287b5-bfd1-4165-b0ed-3776ea0e2727.png)

I'd say the problem is probably in the way your Vue app is setup, using the Vue 3 APIs.

**Comment by catherineluse** [comment]
You're right. It wasn't because of Tailwind. I reproduced it by creating a new Vue 3 app using the Vite installation commands in the official Vue 3 docs (https://v3.vuejs.org/guide/installation.html#vite), then I created a map component in it and the styles were messed up.
